platform/viewer/src/connectedComponents/ConnectedDownloadDialog.js

```diff
@@ -108,7 +109,15 @@ const mapStateToProps = (state, ownProps) => {
         return window.navigator.msSaveBlob(blob, file);
       }
 
-      return cornerstoneTools.SaveAs(viewportElement, file, mimetype);
+      viewportElement.querySelector('canvas').toBlob(blob => {
+        const URLObj = window.URL || window.webkitURL;
+        let a = document.createElement('a');
```

Comment:
`a` is not reassigned. This can probably be `const`

Reply:
Done! thx.